single.just

```diff
@@ -12,6 +12,11 @@ export foundationOwner0 := "0x42d27eEA1AD6e22Af6284F609847CB3Cd56B9c64"
 simulate hdPath='0':
   #!/usr/bin/env bash
   bundlePath=${taskPath}/${bundleName}.json
+  script=SignFromJson
+  if [ -f tasks/eth/006-MCP-L1/SignFromJsonWithAssertions.s.sol ]; then
```

Comment:
This won't work for all possible tasks

We could do something like:

```bash
if [ -f "$PWD/SignFromJson" ]; then
  script="$PWD/SignFromJson"
fi
```

Reply:
This would enforce the requirement that the script is always called `SignFromJson` whether its the root or the task specific script

Reply:
good idea!